skimage/filters/tests/test_thresholding.py

```diff
@@ -703,3 +704,29 @@ def test_multiotsu_lut():
             result = _get_multiotsu_thresh_indices(prob, classes - 1)
 
             assert np.array_equal(result_lut, result)
+
+
+def test_theshold_generalized_histogram():
+    """
+    tests for generalized histogram thresholding algorithm
+    """
+
+    image = data.camera()
+    counts, bin_centers = histogram(image.ravel(), 256, source_range="image")
+
+    # (nu, tau, kappa, omega, threshold)
+    possible_values = [
```

Comment:
I guess you want to [parametrize](https://docs.pytest.org/en/stable/example/parametrize.html) this test :slightly_smiling_face:

Reply:
done